Tasks/ServiceFabricDeployV1/deploy.ps1

```diff
@@ -176,16 +178,22 @@ try {
         $publishParameters['UpgradeParameters'] = $upgradeParameters
         $publishParameters['UnregisterUnusedVersions'] = $unregisterUnusedVersions
         $publishParameters['SkipUpgradeSameTypeAndVersion'] = $skipUpgrade
+        $publishParameters['Status'] = [ref]$status
 
         Publish-UpgradedServiceFabricApplication @publishParameters
     }
     else
     {
         $publishParameters['Action'] = "RegisterAndCreate"
         $publishParameters['OverwriteBehavior'] = Get-VstsInput -Name overwriteBehavior
+        $publishParameters['Status'] = [ref]$status
 
         Publish-NewServiceFabricApplication @publishParameters
     }
-} finally {
+}
+catch {
+    Write-Telemetry "Task_InternalError" "TaskStatus: $status | Exception: $_.Exception.GetType().FullName"
```

Comment:
Remove words "TaskStatus:" and "Exception:". We can use convention to have status and exception type separated by | without any space.

Reply:
Used it for readability. If you don't see any use then lets remove it.

Reply:
It is redundant. As we will never need to see full string. We will always query this data via kusto and it is easier to write query without these extra characters